privacy-on-beam/pbeam/mean_test.go

```diff
@@ -330,10 +330,7 @@ func TestMeanPerKeyWithPartitionsAddsNoiseFloat(t *testing.T) {
 		epsilonNoise, deltaNoise := tc.epsilon, tc.delta
 		k := 23.0
 		l0Sensitivity, lInfSensitivity := 1.0, 2.0
```

Comment:
lInfSensitivity should be 3.0, since lower is 0 and upper is 3.0

And below, when we are calling MeanPerKey, we should use lower and upper instead of hardcoded 0.0 and 2.0 for MinValue and MaxValue.

(Also for the test w/o partitions above, this seems to be an existing bug)